src/reference/modules/administration/pages/relayd.adoc

```diff
@@ -0,0 +1,61 @@
+= Relayd administration
+
+The `rudder-relayd` service (added in 6.0) is part of the `rudder-server-relay` package
+and runs on all relays and root servers. It provides the https://docs.rudder.io/api/relay/[relay API]
+and handles inventories and reports forwarding to the root server from nodes.
+
+== Configuration
+
+Relay configuration is stored in `/opt/rudder/etc/relay`:
+
+* `main.conf` contains general configuration. Its content is configured by the Rudder agent of the relay.
+* `logging.conf` contains logging information.
+
+Relay also reads the following data files:
+
+* `/var/rudder/lib/relay/nodeslist.json`: contains information about the nodes behind a relay
+* `/var/rudder/lib/ssl/allnodescerts.pem` (only on root server): contains the certificates of all nodes managed by the root server (only on root server)
+
+Configuration reload uses the reload API endpoint, which allows `systemctl reload rudder-relayd`
+to be synchronous (and ensure configuration is already applied when command returns).
+
+Reloading relayd configuration reloads all data files and the `logging.conf` file,
```

Comment:
I needed several pass to understand that sentence, I wasn't understanding the opposition introduced by the comma. Perhaps:
```
Reloading relayd configuration reloads all data files and update properties from
`logging.conf` but not the one from `main.conf`. Changing properties in `main.conf`
requires a restart to be taken into account. 
```